### DISCUSSION TITLE
We are seeing dangling index after dropping a table.

### DISCUSSION THREAD

**Discussion by nazar200** [crate/crate#11619] [Q&A]
Hello Everyone, 

We are seeing o.e.g.DanglingIndices after dropping few tables from Crate and we see many shards in unassinged state.

stdout.log:""2021-07-15 10:04:17 [2021-07-15T10:04:17,446][INFO ][o.e.g.DanglingIndicesState] [] [[doc..partitioned.detected_user.04732dhg64qj2c1k60o30c1g/NYtx6xcTRM2Gf2LXueH0KQ]] dangling index exists on local file system, but not in cluster metadata, auto import to cluster state.

Any option to clean these shards. But in Admin UI I don't see the table and cluster is in GREEN.

**Comment by jayeff** [comment]
This is a crosspost to community.crate.io. Please follow the thread there for updates

https://community.crate.io/t/we-are-seeing-dangling-index-after-dropping-a-table/760/2
